raft/src/main/java/org/apache/kafka/raft/ElectionState.java

```diff
@@ -18,6 +18,8 @@
 
 import java.util.OptionalInt;
 
+import static org.apache.kafka.raft.FileBasedStateStore.UNDEFINED_EPOCH;
```

Comment:
nit: it's a little weird to depend on `FileBasedStateStore`. Should we move `UNDEFINED_EPOCH` to `QuorumStateStore`?